custom_components/tuya_local/devices/probreeze_pb20.yaml

```diff
@@ -0,0 +1,165 @@
+name: Pro Breeze 30L Dehumidifier
```

Comment:
Just use a generic name here, in case other brands match.  The products section has the branding details (and one day will be used for matching products)